rpc/codec/server.go

```diff
@@ -6,60 +6,44 @@ package codec
 
 import (
 	"bufio"
-	"errors"
+	"bytes"
```

Comment:
It's time to adjust the copyright for files in this package which you've rewritten. Why don't you add our copyright notice on top and move the original copyright beneath it with a one line space between them?